driver/esp32/espidf.c

```diff
@@ -187,24 +189,43 @@ void ili9xxx_flush(void *_disp_drv, const void *_area, void *_color_p)
 	mp_get_buffer_raise(mp_obj_dict_get(disp_drv->user_data, MP_OBJ_NEW_QSTR(MP_QSTR_spi)), &buffer_info, MP_BUFFER_READ);
 	spi_device_handle_t *spi_ptr = buffer_info.buf;
 
+    int x1 = area->x1;
+    int x2 = area->x2;
+    int y1 = area->y1;
+    int y2 = area->y2;
+
+    // st7789 may need a start_x and start_y offset for display smaller then 320x240 like the TTGO T-Display.
+    if (dt == DISPLAY_TYPE_ST7789) {
+        int start_x = mp_obj_get_int(mp_obj_dict_get(disp_drv->user_data, MP_OBJ_NEW_QSTR(MP_QSTR_start_x)));
+        if (start_x) {
```

Comment:
Do you really need `if (start_x)`, `if (start_y)`?
If they are 0 then adding 0 is cheaper than if statement

Reply:
That makes sense. Will do.